README.md

```diff
@@ -10,4 +11,7 @@ This project aim to deal with modules related to manage Odoo server environment
  - Manage configuration depending on environment (devs, test, prod,..)
  - Keep the security on update
  - Manage email settings
- -...
+
+Translation Status
+------------------
+[![Transifex Status](https://www.transifex.com/projects/p/OCA-server-tools-8-0/chart/image_png)](https://www.transifex.com/projects/p/OCA-server-tools-8-0)
```

Comment:
This happens until the first commit, so we should check it knowing that it will be this way :wink: